src/index.tsx

```diff
@@ -390,6 +390,23 @@ function handlerRectangles(element: ExcalidrawElement, sceneState: SceneState) {
     8
   ]; // se
 
+  if (element.type === "arrow") {
+    if (
+      (element.width < 0 && element.height < 0) ||
+      (element.width > 0 && element.height > 0)
+    ) {
+      return {
+        se: handlers.se,
+        nw: handlers.nw
+      };
+    }
+
+    return {
```

Comment:
Aren't those two return statements the same? The properties on the left in the object literal are swapped, but so are the expressions on the right.

Reply:
Yes, thanks! I'l fix it